backup/backup.go

```diff
@@ -188,6 +188,181 @@ func (c *BackupCommand) BackupInfo(cliConnection plugin.CliConnection, backupId
+
+	if helper.GetAccessToken(helper.ReadConfigJsonFile()) == "" {
+		errors.NoAccessTokenError("Access Token")
+	}
+
+	client := GetHttpClient()
+
+	var userSpaceGuid string = helper.GetSpaceGUID(helper.ReadConfigJsonFile())
+
+	var guid string = serviceInstanceGuid
+	var ServiceName string = guidTranslator.FindInstanceName(cliConnection, guid, nil)
+
+	if ServiceName == "" {
+		errors.IncorrectGuid(guid)
+		return
+	}
+
+	var apiEndpoint string = helper.GetApiEndpoint(helper.ReadConfigJsonFile())
+	var broker string = GetBrokerName()
+	var extUrl string = GetExtUrl()
+
+	apiEndpoint = strings.Replace(apiEndpoint, "api", broker, 1)
+
+	req, err := http.NewRequest("GET", apiEndpoint+extUrl+"/backups"+"?space_guid="+userSpaceGuid, nil)
+	errors.ErrorIsNil(err)
+
+	var resp *http.Response = GetResponse(client, req)
+
+	defer resp.Body.Close()
+
+	body, err := ioutil.ReadAll(resp.Body)
+
+	table := tablewriter.NewWriter(os.Stdout)
+	table.SetBorder(false)
+	table.SetHeaderAlignment(tablewriter.ALIGN_LEFT)
+	table.SetAlignment(tablewriter.ALIGN_LEFT)
+	table.SetCenterSeparator(" ")
+	table.SetColumnSeparator(" ")
+	table.SetRowSeparator(" ")
+	table.SetHeaderLine(false)
+	table.SetAutoFormatHeaders(false)
+
+	if resp.Status == "200 OK" {
+		fmt.Println(AddColor("OK", green))
+		//var temp = string(body)
+
+		//var lines = strings.Split(temp, ",")
+		var flag bool
+
+		table.SetHeader([]string{AddColor("backup_guid", white), AddColor("username", white), AddColor("type", white), AddColor("trigger", white), AddColor("started_at", white), AddColor("finished_at", white)})
+
+		var response []interface{}
+		if err := json.Unmarshal(body, &response); err != nil {
+			fmt.Println("Invalid response for the request ", err)
+		}
+
+		var no_of_columns int = 6
+		var field = make([]string, no_of_columns)
+
+		for backup := range response {
+			if strings.Contains(guid, (response[backup].(map[string]interface{}))["instance_guid"].(string)) {
+				field[1] = (response[backup].(map[string]interface{}))["username"].(string)
+				field[2] = (response[backup].(map[string]interface{}))["type"].(string)
+				field[0] = (response[backup].(map[string]interface{}))["backup_guid"].(string)
+				field[0] = AddColor(field[0], cyan)
+				field[3] = (response[backup].(map[string]interface{}))["trigger"].(string)
+				field[4] = (response[backup].(map[string]interface{}))["started_at"].(string)
+				field[5], flag = (response[backup].(map[string]interface{}))["finished_at"].(string)
+				if flag == false {
+					field[5] = "null"
+				}
+				table.Append(field)
+			}
+		}
+
+	}
+	table.Render()
+	if resp.Status != "200 OK" {
+		fmt.Println(AddColor("FAILED", red))
+		var message string = string(body)
+		var parts []string = strings.Split(message, ":")
+		fmt.Println(parts[2])
+	}
```

Comment:
This whole block of rendering the table [L225](https://github.com/cloudfoundry-incubator/service-fabrik-cli-plugin/pull/2/files#diff-c1200ceabb9994d8786d2389fae289c1R225)-[L275](https://github.com/cloudfoundry-incubator/service-fabrik-cli-plugin/pull/2/files#diff-c1200ceabb9994d8786d2389fae289c1R275) appears in all the list backups functions (ListDeletedBackupsByInstanceName, ListBackupsByInstanceName, ListBackups, ListBackupsByInstanceGuid). Is it not possible to refactor it in a single function and use it everywhere, lot of code duplicated.